util/regression-tests/tests/REQUEST-941-APPLICATION-ATTACK-XSS/941130.yaml

```diff
@@ -1,23 +1,278 @@
 ---
-  meta: 
-    author: "csanders-git"
-    enabled: true
-    name: "941130.yaml"
+  meta:
+    author: "csanders-git, Christian Folini"
     description: "Tests to trigger, or not trigger 941130"
-  tests: 
-    - 
-      test_title: 941130-1
-      desc: XSS in XML Test as described in http://www.client9.com/article/five-interesting-injection-attacks/
-      stages:
-      - 
-        stage:
-          input:
-            dest_addr: 127.0.0.1
-            method: GET
-            port: 80
-            uri: '/char_test?mime=text/xml&body=%3Cx:script%20xmlns:x=%22http://www.w3.org/1999/xhtml%22%20src=%22data:,alert(1)%22%20/%3E'
-            headers:
-              User-Agent: ModSecurity CRS 3 Tests
-              Host: localhost
-          output:
-            log_contains: id "941130"                
\ No newline at end of file
+    enabled: true
+    name: 941130.yaml
+  tests:
+  -
+    test_title: 941130-1
+    desc: XSS in XML Test as described in http://www.client9.com/article/five-interesting-injection-attacks/
+    stages:
+    - 
+      stage:
+        input:
+          dest_addr: 127.0.0.1
+          method: GET
+          port: 80
+          uri: '/char_test?mime=text/xml&body=%3Cx:script%20xmlns:x=%22http://www.w3.org/1999/xhtml%22%20src=%22data:,alert(1)%22%20/%3E'
+          headers:
+            User-Agent: ModSecurity CRS 3 Tests
+            Host: localhost
+        output:
+          log_contains: id "941130"
+  - 
+    test_title: 941130-2
+    desc: "XSS test for 941130"
+    stages:
+    - 
+      stage:
+        input:
+          dest_addr: 127.0.0.1
+          headers:
+            Host: localhost
+            method: POST
+            port: 80
```

Comment:
port should be part of input, not headers.

Reply:
method too.

Reply:
Can we also ensure that we have User-Agent and maybe Accept too so ideally we're only tripping a single rule?